src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -39,7 +39,7 @@ def dialog_content
         VBox(
           alternative_information,
           create_alternatives_table,
-          RichText(Id(:slaves), _("Please select an alternative to view his slaves.")),
+          RichText(Id(:slaves), "<pre>#{@alternative.choices.first.slaves}</pre>"),
```

Comment:
I though in reuse `alternatives_handler` but this method do a query to the table that not exists yet, so if I try to execute `alternatives_handler` in the `dialog_content` or `initialize` it does not work.
And to avoid repeat `<pre>#{whatever.slaves}</pre>` I could create a method that only do that and use it in both sites.